frontend/src/styles/LandingPage.css

```diff
@@ -0,0 +1,50 @@
+.LandingPage main {
+  background: var(--primary-bg-color);
+  padding: 38px 35px;
+}
+.card .card-header {
+  background: var(--primary-bg-color);
+  border: 0;
+  padding-left: 0;
+  padding-right: 0;
+}
+
+.accordion .card {
+  border: none !important;
+  background-color: transparent;
+}
+.card .card-body {
+  background-color: var(--secondary-light-color);
+  padding: 0.25rem;
+  margin-bottom: 30px;
+}
+.form-wrapper {
+  width: 100%;
+  text-align: right;
+}
+
+.form-group {
+  font-size: 20px;
+  display: flex;
+}
+.form-group .form-label {
+  text-align: left;
+  padding: 45px 10px 25px 10px;
+}
+.form-group input {
+  background-color: var(--input-bg-color);
```

Comment:
Variable input-bg-color is not defined anywhere. Did you forget to add changes in App.css?

Reply:
i forgot to push app.cc file earlier. But i have pushed it now. you can review it now.

Reply:
Resolved in 5f37977c717e71dfe2cbcf56d4668acac47e6947